voluncheer/opportunityboard/models.py

```diff
@@ -80,6 +85,9 @@ class Meta:
     description = models.TextField()
     date = models.DateTimeField()
     end = models.TimeField()
+    is_recurring = models.BooleanField(default=False)
+    recurrence = models.CharField(max_length=6, choices=FREQUENCIES, null=True, blank=True)
```

Comment:
Are you reusing FREQUENCIES anywhere else?

Reply:
Moved `FREQUENCIES `under Opportunity model definition